### ISSUE TITLE
keyboard help mode for screen reader users 

### ISSUE THREAD

**Issue by adil-s** [microsoft/vscode#73114]
thanks to you all for making this IDE accessible for screen reader users, 
quick command panel always helps me out to find out keyboard shortcuts. 
but I would like to have some sort of keyboard help mode where I can just press keyboard shortcuts and I should hear assigned action to key stroke. 
I. E.  when I press CTRL +  F4, it should say, "closes the current document". 
first, I should be able to turn on the help mode by pressing some key combination and try out the different key shortcuts and turn off the help mode. 
it can help me a lot when I have forgotten the use of any particular keyboard shortcut.

**Comment by alexandrudima** [comment]
@adil-s You could open the default keybindings (which are in a text format) and search in file for the keybinding you'd want to try. These are available via `F1 > Preferences: Open Default Keyboard Shortcuts (JSON)`

**Comment by adil-s** [comment]
yes, it does help and I have been using that method to discover new keyboard shortcuts. but it doesn't solve actual problem. the default keyboard JSOn file contains more than thousand keyboard shortcuts and finding desired keyboard in JSON file feels a bit cumbersome using find dialogue box. 
the reason for this proposal was to make experience better for screen reader users who are new to VS code.  
I think, initially, they would have no idea about default keyboard shortcuts JSON file.
the another approach could be to have some sort of getting started guide for screen reader users in which there is a little walk through of VS code explaining how to access the different part of window (menu bar, tool bar notification, jumping back to editor etc).
because from a screen reader perspective VS code is just a browser like application. I've still not found keyboard shortcut to jump to tool bar in VS code. 
in standard desktop application, I can press ALT and then press CTRL+TAB to go to tool bar. but it doesn't work that way in Vs code. 
as of now, I use standard web navigation keyboard shorctus (hfor headings, B for buttons,E for edit box etc) for to navigate VS code windows. but it reduces my productivity.

**Comment by Neurrone** [comment]
Its probably still better to be able to search the bindings like how anyone else would do it. 

The proposed UX would only make sense for a screen reader user, which is why its found nowhere else.

**Comment by alexandrudima** [comment]
There is a more interactive UI for keyboard shortcuts, the shortcuts editor, available via ctrl+k ctrl+s. Could you perhaps give that a try? 

Other places to find out about keyboard shortcuts would be the top level menu or the quick command palette (which you have already found). There is also some cheat-sheat PDFs like e.g. https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf

I like the idea of a getting started guide aimed towards screen reader users. @gregvanl is in charge of the vscode-docs and we are already have an accessibility focused page at https://code.visualstudio.com/docs/editor/accessibility . Perhaps that page could be improved ?

VS Code doesn't have a toolbar like other classic IDEs, it only has menus, the sidebar (with a possible viewlet), the main editor area and the status bar.

**Comment by Neurrone** [comment]
I'll be willing to take a look at that page and add to it as needed in the next couple of weeks.

**Comment by adil-s** [comment]
I'm glad that you're willing to work on accessibility. 
thank you!
please add the  cheat sheet on accessibility page.
